### PR TITLE
Create podcast blogpost

### PR DESCRIPTION
**Pull request by AlbaRoza**
Launch of the Foundation for Public Code new podcast.

-----
[View rendered _posts/2020-09-01-on-air-next-our-podcast.md](https://github.com/publiccodenet/blog/blob/AlbaRoza-patch-2-1/_posts/2020-09-01-on-air-next-our-podcast.md)

### CODE REVIEW

File: _posts/2020-09-01-on-air-next-our-podcast.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title:  "On air next: our podcast!"
+date: 2020-09-01
+author: Alba Roza
+type: blogpost
+excerpt: Launch of the livestream/videopodcast project
```
Comment (ElenaFdR) [inline _posts/2020-09-01-on-air-next-our-podcast.md:6]: To get the name everywhere the excerpt goes :)

```suggestion
excerpt: We're launching a new livestream/video podcast: 'Let's talk about public code'
```

File: _posts/2020-09-01-on-air-next-our-podcast.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+# On air next: our podcast!
+
+A community couldn't exist without knowledge sharing. Indeed, this is one of its most powerful and appealing characteristics, although it's obvious that not all of its members can invest the same number of hours collaborating in a project. 
+
+Sharing assets is something we always keep in mind at the Foundation for Public Code. We've always tried to share as many guides, tools and tips as we can in order to nurture our community. However there was something we were lacking, an even more direct way of communicating with our community: a live, closer and bilateral communication.
+
+This is why we are **launching our first live stream/podcast, 'Let's talk about public code'**, a space in which we'll interview people strongly related to popular public and open codebases, 'public code' ones.
+
+![Listen to our podcast in our Youtube channel]({{site.url}}/assets/podcast-available-youtube-channel.jpg)
```
Comment (ElenaFdR) [inline _posts/2020-09-01-on-air-next-our-podcast.md:20]: Shouldn't this link to one/all of the livestream platforms, so people can pick their fav? That way it makes more sense to say later on that you can catch it on YouTube if you didn't make it.
(I assume most people consider YouTube to be for pre-recorded content, so we should make extra clear where/how people can view the live version.)

File: _posts/2020-09-01-on-air-next-our-podcast.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## What? When? How?!
+
+Stay tuned next **Thursday, 8 September at 14.00 (UTC)** for the first episode. You will be able to ask real time questions of our first guest, Dr. Arnau Monterde, coordinator from the popular project [Decidim](https://decidim.org/), a public codebase that has been replicated in dozens of organizations in Spain, Canada, Finland, Mexico, France, Sweden and Italy.
+
+Think of that issue your codebase is stumbling upon, that documentation question you always have in mind, that product backlog that is stuck... what if you could ask a real expert about it? Come along and [watch our YouTube channel](https://www.youtube.com/channel/UCXIL94kkenw0cs_ZgNhKYuw) next Thursday, we'll be waiting for you!
```
Comment (ElenaFdR) [inline _posts/2020-09-01-on-air-next-our-podcast.md:26]:
```suggestion
Think of that issue your codebase is stumbling upon, that documentation question you always have in mind, that product backlog that is stuck... what if you could ask a real expert about it? Bring all your questions to our livestream next Thursday, we'll be waiting for you!
```

**Review by ElenaFdR — COMMENTED** [review]
This looks good! I've made a few suggestions.
And it's missing a link to the photo.

File: _posts/2020-09-01-on-air-next-our-podcast.md (hunk 2 above)
Comment (AlbaRoza) [reply _posts/2020-09-01-on-air-next-our-podcast.md:20]: You're right. Let's add it in the end of the blog post.

File: _posts/2020-09-01-on-air-next-our-podcast.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+![Listen to our podcast in our Youtube channel]({{site.url}}/assets/podcast-available-youtube-channel.jpg)
+
+## What? When? How?!
+
+Stay tuned next **Thursday, 8 September at 14.00 (UTC)** for the first episode. You will be able to ask real time questions of our first guest, Dr. Arnau Monterde, coordinator from the popular project [Decidim](https://decidim.org/), a public codebase that has been replicated in dozens of organizations in Spain, Canada, Finland, Mexico, France, Sweden and Italy.
+
+Think of that issue your codebase is stumbling upon, that documentation question you always have in mind, that product backlog that is stuck... what if you could ask a real expert about it? Bring all your questions to our livestream next Thursday, we'll be waiting for you! We are going to be live in several platforms like [Youtube](https://www.youtube.com/channel/UCXIL94kkenw0cs_ZgNhKYuw), [Periscope](https://twitter.com/publiccodenet), [Facebook](https://www.facebook.com/publiccodenet) and [Linkedin](https://www.linkedin.com/company/publiccodenet/), choose your favourite one and join the conversation!
```
Comment (AlbaRoza) [inline _posts/2020-09-01-on-air-next-our-podcast.md:26]: I've added the different channels here, @ElenaFdR

**Review by AlbaRoza — COMMENTED** [review]
Added the social media channels at the end of the blogpost

**Comment by ElenaFdR** [comment]
LinkedIn links always break Travis - this is because LinkedIn puts a login screen in front of content in such a way that Travis can't confirm the URL exists.

File: _posts/2020-09-01-on-air-next-our-podcast.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+Think of that issue your codebase is stumbling upon, that documentation question you always have in mind, that product backlog that is stuck... what if you could ask a real expert about it? Bring all your questions to our livestream next Thursday, we'll be waiting for you! We are going to be live in several platforms like [Youtube](https://www.youtube.com/channel/UCXIL94kkenw0cs_ZgNhKYuw), [Periscope](https://twitter.com/publiccodenet), [Facebook](https://www.facebook.com/publiccodenet) and [Linkedin](https://www.linkedin.com/company/publiccodenet/), choose your favourite one and join the conversation! 
+
+Are you unable to join then? Don't worry, we've also thought about that. We are going to publish the interview in [podcast format](https://podcast.publiccode.net/) once it's edited, so everyone has the chance of listening to Arnau's insights.
```
Comment (ElenaFdR) [inline _posts/2020-09-01-on-air-next-our-podcast.md:28]:
```suggestion
Are you unable to join then? Don't worry, we've also thought about that. We'll publish the interview in [podcast format](https://podcast.publiccode.net/) once it's edited, so everyone has the chance of listening to Arnau's insights.
```

**Review by AlbaRoza — COMMENTED** [review]
Changes made (Linkedin and Elena's suggestion)

**Pull request merged**